app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
tìm hiểu và sử dụng I18n cho TOÀN BỘ TEXT trong TẤT CẢ các file HTML nhé

`Ruby on Rails Tutorial Sample App`